go.mod

```diff
@@ -0,0 +1,13 @@
+module gopkg.in/DataDog/dd-trace-go.v1
+
+go 1.12
+
+require (
+	github.com/DataDog/datadog-go v3.7.1+incompatible
+	github.com/philhofer/fwd v1.0.0 // indirect
+	github.com/tinylib/msgp v1.1.2
+	github.com/zenazn/goji v1.0.1
```

Comment:
Do we want goji here?

Reply:
It was an accident. There should be only one entry in this file: `libp/msgp`

Reply:
I think this is going to be weird. It requires doing:
```
git update-index --assume-unchanged ./go.mod 
```
Because the file exists in the repo :|